lua/telescope/builtin/git.lua

```diff
@@ -98,26 +92,42 @@ git.branches = function(opts)
     upstream = 0,
     committerdate = 0,
   }
-  local register_entry = function(entry, trim_refname_prefix)
-    entry.name = string.sub(entry.refname, string.len(trim_refname_prefix)+1)
+  local unescape_single_quote = function(v)
+      return string.gsub(v, "\\([\\'])", "%1")
+  end
+  local parse_line = function(line)
+    local fields = vim.fn.split(string.sub(line, 2, -2), "''", true)
```

Comment:
probably better to use `vim.split` instead? I'd avoid `vim.fn.*` when possible :)

Reply:
thanks! I fixed it.